src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -62,7 +60,7 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
   }, [toggleSidebar]);
 
   const goToAdmin = useCallback(() => {
-    openUrl("/admin");
+    openUrlInNewTab(`${window.location.origin}/admin`);
```

Comment:
Why new tab?

Reply:
Ed Cook asked for that specifically. I've just included it in this PR to speed things up.

Reply:
@dmitrymarchuk Could be useful if mentioned in the PR description, with possible links to source. That eliminates questions like this and possible return to re-review